lib/data/data_sources/remote_data_source/models/sendbird_chat_member.dart

```diff
@@ -0,0 +1,10 @@
+import 'package:sendbird_sdk/core/models/member.dart';
+import 'package:sendbirdtutorial/domain/entities/chat_user.dart';
+
+extension MemberExtension on Member {
```

Comment:
let's use the file name as the model name, so in this case this file should be member.dart, I prefer not to have the sendbird brand/name in the files names